index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+      <header id="header">
+        <img id="header-img" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"></img>
+
+        <nav id="nav-bar">
+          <ul>
+            <li><a class="nav-link" href="#section1">Features</a></li>
+            <li><a class="nav-link"  href="#section2">How It Works</a></li>
+            <li><a class="nav-link"  href="#section3">Pricing</a></li>
+          </ul>
+        </nav>
+      </header>
+
+
+
+      <section id="hero">
+        <h2>Handcrafted, home-made masterpieces</h2>
+        <form id="form" action="https://www.freecodecamp.com/email-submit">
+          <input name="email" id="email" type="email" placeholder="Enter your email address" required="">
+          <input id="submit" type="submit" value="Get Started" class="btn">
+        </form>
+     </section>
+     <table>
+
+      <tr>
+        <section id="section1">
+          <h2>Premium Materials</h2>
+           <p>
+             Our trombones use the shiniest brass which is sourced locally.
+             This will increase the longevity of your purchase.
+           </p>
+        </section>
+      </tr>
+      <tr>
+        <section id="section2">
+          <h2>Fast Shipping</h2>
+          <p>
+            We make sure you recieve your trombone as soon as we have
+            finished making it. We also provide free returns if you are not
+            satisfied.
+          </p>
+        </section>
+      </tr>
+      <tr>
+        <section id="section3">
+          <h2>Fast Shipping</h2>
+          <p>
+            We make sure you recieve your trombone as soon as we have
+            finished making it. We also provide free returns if you are not
+            satisfied.
+          </p>
+        </section>
+      </tr>
+    </table>
+
+
+
+
+
+
+      <section id="sectionVideo">
+        <iframe id="video" src="https://www.youtube-nocookie.com/embed/y8Yv4pnO7qc?rel=0&amp;controls=0&amp;showinfo=0" allowfullscreen="" height="315" frameborder="0"></iframe>
+      </section>
+
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```

Comment:
In index.html : 

Kindly remove the comments to improve your code.

Reply:
You have done a great job stating why you are requesting a change, but I think you can make your reason more detailed😉 It could be cool if you stated how removing comments will improve the code.  remember it is not just about stating why the changes are requested but also making the recipient understand why the changes is requested.😉  for example, removing commented block of code in your code makes your code cleaner and gives it a more professional look😉 Kindly make your reasons for requesting a change more detailed👍